src/pages/AboutPage.js

```diff
@@ -108,32 +108,36 @@ const AboutPage = () => (
       </Col>
       <Col>
         <h2>Contact Us</h2>
-        <p>
-        Want to get involved, or have questions, thoughts, feedback, or concerns?
-        </p>
-        Please use one of the following subject line patterns in your emails:<br/>
-        "Volunteer: [Question or Feedback] [Subject]"<br/>
-        "Project Request: [Question or Feedback] [Subject]"<br/>
-        <br/>
-        Email us at:&nbsp;
-        {/* FIXME update */}
+        <p>Want to get involved, or have questions, thoughts, feedback, or concerns?</p>
+        Please use one of the following subject line patterns in your emails:
+        <br />
+        "Volunteer: [Question or Feedback] [Subject]"
+        <br />
+        "Project Request: [Question or Feedback] [Subject]"
+        <br />
+        <br />
+        Email us at: {/* FIXME update */}
```

Comment:
do we still need the fixme comment?

Reply:
had removed them in the other PR. merging dev into this branch would probably fix it